index.js

```diff
@@ -0,0 +1,136 @@
+const body = document.querySelector('body');
```

Comment:
dont find body, use document.body

Reply:
please, firstly create all elements and secondly put all elements using only one command document.body.appendChild(wrapper)

Reply:
do you mean
const body = document.body;
body.appendChild(someEl);
body.appendChild(anotherEl);?

Reply:
You can save in const, but also you can use 
document.body.appendChild(someEl);
document.body.appendChild(anotherEl);
since we have one body in page, so const and document.body dont find body in dom, they have link to body. One plus: body.appendChild(someEl) is shorter than document.body.appendChild(wrapper)

Reply:
got it
done